### ISSUE TITLE
Wiki

### ISSUE THREAD

**Issue by HHS81** [HHS81/do328#5]
write short wiki entry here, and update FGFS-wiki

**Comment by xcvb85** [comment]
I found a very interesting website:
http://www.smartcockpit.com/plane/DORNIER/DORNIER%20328-JET.html
Maybe we can add it to the documentation sites.

**Comment by HHS81** [comment]
I agree,  We should add it since I already used it as reference for certain things. I forgot about that.
